docs-v2/integrations/all/marketo.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Marketo
+sidebarTitle: Marketo
+---
+
+API configurations: [`marketo`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [OAuth-related docs](https://developers.marketo.com/rest-api/authentication)
+-   [Marketo REST API docs](https://developers.marketo.com/rest-api)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+-   Marketo offers client credentials token that authorizes as app instead of a user.
+-   Both `endpointURL` and `identityURL` can be found in the Admin > Web Services menu in the REST API section.
```

Comment:
Can we add a direct link to this marketo admin page?

Reply:
Added reference to the base URL section since I couldn’t find a direct link in the provider’s documentation.